### ISSUE TITLE
Changing GuestOS fails

### ISSUE THREAD

**Issue by rzubin** [vmware/govmomi#841]
I am trying to change the guestOS from "Other" to either centOS or RHEL.
```
govc vm.change -vm=packer-05 -g=centos
govc: A specified parameter was not correct: configSpec.guestId
```
I have also tried the following
```
centos-64
rhel7-64
```
Does govc take a different format to specify the guestOS?

Thanks 

Zubin

**Comment by dougm** [comment]
See: http://pubs.vmware.com/vsphere-6-5/index.jsp#com.vmware.wssdk.apiref.doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html

You can use "centosGuest" for example.

**Comment by rzubin** [comment]
Hey @dougm thanks! It works now

**Comment by dougm** [comment]
ok great, leaving this open as a reminder to add that to the vm.change usage text.

**Comment by ardrigh** [comment]
I just ran into this same question. I couldn't even find a reference to the API document in the source code.

Now I have used "centos7_64Guest" it works. 

It's not something easily guessable as the correct name to use. So it would be good for some kind of documentation update, perhaps in USAGE ?

**Comment by dougm** [comment]
@ardrigh yes, I have a branch with various doc updates including this, it'll be in the `vm.create -h` text + USAGE.md

**Comment by dougm** [comment]
Included in 7d879ba
